Default - Additional (stricter) performance improvements (#32782)
demisto/content: Packs/DefaultPlaybook/ReleaseNotes/2_0_9.md

#### Playbooks

##### Default

The playbook has been adjusted for improved performance.
- Added a new input that determines whether advanced investigation steps will be executed.

> - Inline indicator extraction no longer runs by default. It requires the ExtractIndicators and the new AdvancedInvestigation playbook inputs to both have the value of True. This will ensure that inline extraction occurs only if the subsequent playbook tasks need it to function correctly.

Reviewer: ```suggestion
- Updated the inline indicator extraction to no longer run by default. It requires the ExtractIndicators and the new AdvancedInvestigation playbook inputs to both have the value of True. This will ensure that inline extraction occurs only if the subsequent playbook tasks need it to function correctly.
```

> - Additional hashes that were extracted from files will not be searched across other endpoints. Original incident file attachments will be searched in the Advanced Investigation flow.

Reviewer: ```suggestion
- Changed the flow so that additional hashes that were extracted from files will not be searched across other endpoints. Original incident file attachments will be searched in the Advanced Investigation flow.
```

- Changed the maximum images that can be extracted from PDF files to 2.

> - Fixed an issue where the Investigation tab of the layout did not show the `Selected Indicators` field, because it was not setting an initial value in it, and that layout tab is configured to hide empty fields.

Reviewer: ```suggestion
- Fixed an issue where the Investigation tab of the layout did not show the `Selected Indicators` field, because it was not setting an initial value in it, and that layout tab was configured to hide empty fields.
```
